src/middleware/api-documentation-link.ts

```diff
@@ -1,14 +1,13 @@
 import formatLinkHeader from 'format-link-header';
 import { Context, Middleware, Next } from 'koa';
-import { hydra } from 'rdf-namespaces';
 import url from 'url';
 
 export default (path: string): Middleware => (
   async ({ request, response }: Context, next: Next): Promise<void> => {
     await next();
 
     const link = {
-      rel: hydra.apiDocumentation,
+      rel: 'http://www.w3.org/ns/hydra/core#apiDocumentation',
```

Comment:
Clearly not RDF as it ends up into a `Link` header. Also not listed in https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types so assuming it's the Hydra specification dictating it.